compiler/arser/include/arser/arser.h

```diff
@@ -33,6 +33,7 @@
 #include <cstring>
 
 #include <cassert>
+#include <cstdint>
```

Comment:
Just information:
This is related to https://gcc.gnu.org/gcc-13/porting_to.html#Header_dependency_changes